templates/essentials.yaml

```diff
@@ -429,6 +429,21 @@ Resources:
                   - "ec2:CreateTags"
                 Resource:
                   - 'arn:aws:ec2:*::snapshot/*'
+  AWSSSMManagedInstanceRole:
```

Comment:
don't we want to create a `ManagedInstanceProfile` and associate this role with the profile?